lib/ansible/module_utils/redfish_utils.py

```diff
@@ -656,12 +673,15 @@ def set_bios_default_settings(self):
             return response
         return {'ret': True, 'changed': True, 'msg': "Set BIOS to default settings"}
 
-    def set_one_time_boot_device(self, bootdevice):
+    def set_multi_bios_default_settings(self):
```

Comment:
This "multi" method isn't needed in this PR since this is for a "set" method called from redfish_config.

Reply:
Done